### DISCUSSION TITLE
Pyvistaqt export as .vtkjs

### DISCUSSION THREAD

**Discussion by Jonas231** [pyvista/pyvista#1407] [Q&A] [answered]
Hello, 
I am plotting points and lines in the background plotter. 
I use these options:
```py
plotter2.add_bounding_box()
plotter2.add_axes()
#plotter2.add_bounds_axes()
plotter2.show_bounds(font_size = 15, color='white')
plotter2.set_scale(xscale=1, zscale= 1)
plotter2.add_legend(size = [0.2, 0.2])
```
When I export as .vtkjs only the points and lines (my data) and the bounding box are shown in the 
http://viewer.pyvista.org/, i.e. the bounds and the legend are missing.

Can something be done about that?
Recently I plotted some points in regular distances just to have a rough clue about the dimensions of my 
data when viewing it as .vtkjs only.

Kind regards,
Jonas

**Comment by akaszynski** [comment, marked as answer]
Sadly, not all values or actors can be exported into vtkjs.  This is likely to be improved in future releases of VTK.

**Reply by akaszynski** [reply]
An additional note for anyone still reading this.  Some issues like these would be best raised on Kitware's VTK gitlab repository https://github.com/Kitware/VTK.  It's completely fine raising issues/discussions here, and we'll do the best we can to answer or redirect you to someone who can.
